metaflow/plugins/aip/interruptible_decorator.py

```diff
@@ -11,7 +11,7 @@ def _get_ec2_metadata(path: str) -> Optional[str]:
     return response.text
 
 
-class interruptibleDecorator(StepDecorator):
+class InterruptibleDecorator(StepDecorator):
```

Comment:
Align with python naming convention. Not a breaking change as the decorator is imported by the `name` variable on line 34.

Reply:
Also, [no customer projects reference this](https://gitlab.zgtools.net/search?group_id=181&scope=blobs&search=interruptibleDecorator), so it won't break anything.